### PR TITLE
Update Signing_Data.md

### PR DESCRIPTION
**Pull request by chy2021**
fix a bug.

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/zh-hans.developer.conflux-chain.org/2)

### CODE REVIEW

File: docs/conflux-portal/API_Reference/Signing_Data.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
 id: signing_data_with_portal
-title: 使用Conflux Portal浏览器插件钱包对数据签名
+title: 使用 Portal 浏览器插件钱包对数据签名
```
Comment (AngeliaQ) [inline docs/conflux-portal/API_Reference/Signing_Data.md]: suggestion:  'Portal' can be replaced with 'ConfluxPortal', thanks for your effort.

File: docs/conflux-portal/API_Reference/Signing_Data.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 custom_edit_url: https://github.com/Conflux-Chain/conflux-portal-docs/edit/master/docs/cn/portal/API_Reference/Signing_Data.md
 ---
-## 签名方法的历史[#](https://developer.conflux-chain.org/docs/introduction/en/conflux_overview/#history-of-the-signing-methods)
+## 签名方法[#](https://developer.conflux-chain.org/docs/introduction/en/conflux_overview/#history-of-the-signing-methods)
 
-Portal是由MetaMask继承而来，MetaMask中有多种签名方法，你可以在MetaMask文档中查阅所有历史签名方法。
+Conflux Portal 是由 MetaMask 继承而来，MetaMask 中有多种签名方法，您可以在 MetaMask 文档中查阅所有签名方法。
```
Comment (AngeliaQ) [inline docs/conflux-portal/API_Reference/Signing_Data.md]: same as above

File: docs/conflux-portal/API_Reference/Signing_Data.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
-## ConfluxPortal中有何改善[#](https://developer.conflux-chain.org/docs/introduction/en/conflux_overview/#what-changed-in-confluxportal)
+## Portal 中有何改善[#](https://developer.conflux-chain.org/docs/introduction/en/conflux_overview/#what-changed-in-confluxportal)
```
Comment (AngeliaQ) [inline docs/conflux-portal/API_Reference/Signing_Data.md]: same as above

File: docs/conflux-portal/API_Reference/Signing_Data.md (hunk 1 above)
Comment (chy2021) [reply docs/conflux-portal/API_Reference/Signing_Data.md]: ok

**Review by chy2021 — COMMENTED** [review]
ok

**Review by AngeliaQ — APPROVED** [review]

**Pull request merged**